Bottom: d899c851f79c918fd38353f03a10579aa99e9dd6
Top:    1677cda502ea3d2ee6fe2e16e11cdc07dfa35bee
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-09-14 23:07:42 +0100

Refresh of undo-c-handle-undo-failures-in

---

diff --git a/src/undo.c b/src/undo.c
index c20d78f..7832600 100644
--- a/src/undo.c
+++ b/src/undo.c
@@ -958,9 +958,9 @@ Undo (bool draw)
 
   if (ptr->Serial > Serial)
     {
-      Messasge (_("ERROR: Bad undo serial number %d in undo stack - expecting %d or lower\n"
-                  "       Please save your work and report this bug.\n"),
-                prt->Serial, Serial);
+      Message (_("ERROR: Bad undo serial number %d in undo stack - expecting %d or lower\n"
+                 "       Please save your work and report this bug.\n"),
+               ptr->Serial, Serial);
 
       /* It is likely that the serial number got corrupted through some bad
        * use of the SaveUndoSerialNumber() / RestoreUndoSerialNumber() APIs.
@@ -987,7 +987,7 @@ Undo (bool draw)
   UnlockUndo ();
 
   if (error_undoing)
-    Messasge (_("ERROR: Failed to undo some operations\n"));
+    Message (_("ERROR: Failed to undo some operations\n"));
 
   if (Types && andDraw)
     Draw ();
@@ -1131,9 +1131,9 @@ Redo (bool draw)
 
   if (ptr->Serial < Serial)
     {
-      Messasge (_("ERROR: Bad undo serial number %d in redo stack - expecting %d or higher\n"
-                  "       Please save your work and report this bug.\n"),
-                prt->Serial, Serial);
+      Message (_("ERROR: Bad undo serial number %d in redo stack - expecting %d or higher\n"
+                 "       Please save your work and report this bug.\n"),
+               ptr->Serial, Serial);
 
       /* It is likely that the serial number got corrupted through some bad
        * use of the SaveUndoSerialNumber() / RestoreUndoSerialNumber() APIs.
@@ -1163,7 +1163,7 @@ Redo (bool draw)
   UnlockUndo ();
 
   if (error_undoing)
-    Messasge (_("ERROR: Failed to redo some operations\n"));
+    Message (_("ERROR: Failed to redo some operations\n"));
 
   if (Types && andDraw)
     Draw ();
